tensorflow/python/keras/saving/save.py

```diff
@@ -18,18 +18,21 @@
 from __future__ import division
 from __future__ import print_function
 
-import os
 import sys
 
 import six
 
-from tensorflow.python import tf2
 from tensorflow.python.keras.saving import hdf5_format
 from tensorflow.python.keras.saving.saved_model import load as saved_model_load
 from tensorflow.python.keras.saving.saved_model import save as saved_model_save
 from tensorflow.python.saved_model import loader_impl
+from tensorflow.python.util.lazy_loader import LazyLoader
 from tensorflow.python.util.tf_export import keras_export
 
+network = LazyLoader(
+    'network', globals(),
+    'tensorflow.python.keras.engine.network')
```

Comment:
There's an internal bug, so this has to use double quotes:
```
# pylint:disable=g-inconsistent-quotes
network = LazyLoader(
    "network", globals(),
    "tensorflow.python.keras.engine.network")
# pylint:enable=g-inconsistent-quotes
```

Reply:
Thanks!

Reply:
Done! :+1: